scrapy/mail.py

```diff
@@ -73,20 +75,19 @@ def send(self, to, subject, body, cc=None, attachs=(), mimetype='text/plain', ch
             rcpts.extend(cc)
             msg['Cc'] = COMMASPACE.join(cc)
 
-        if charset:
-            msg.set_charset(charset)
```

Comment:
Why are you not calling `msg.set_charset` anymore when there are no attachments?